labels.yml

```diff
@@ -0,0 +1,110 @@
+  color: "#0052cc"
+  description: "Affects the documentation."
+  aliases: [documentation]
+- name: Duplicate
+  color: "#cfd3d7"
+  description: "This issue or pull request already exists."
+  aliases: [duplicate]
+# Effort
+- name: Effort High
+  color: "#d4c5f9"
+  description: "Requires a large time investment."
+- name: Effort Medium
+  color: "#d4c5f9"
+  description: "Requires a moderate time investment."
+- name: Effort Low
+  color: "#d4c5f9"
+  description: "Requires a small time investment."
+- name: Feature Request
+  color: "#009800"
+  description: "New feature wanted!"
+  aliases: [enhancement]
+- name: Good First Issue
+  color: "#0d8387"
+  description: "The best issues for new people to tackle!"
+  aliases: [good first issue]
+- name: Infrastructure
+  color: "#6eb1c9"
+  description: "Issues or PRs that affect the CI or packaging."
+- name: Invalid
+  color: "#000000"
+  description: "For issues and PRs which are not valid contributions."
+- name: Keep Open
+  color: "#ffffff"
+  description: "A label to ward off stale bots."
+- name: Merge When CI Passes
+  color: "#0e8a16"
+  description: "Hit that merge button when it's all green!"
+- name: Needs Review
+  color: "#2E143F"
+  description: "Needs reviews before merge."
+- name: No Changelog Entry Needed
+  color: "#e99695"
+  description: "Skip any changelog checks."
+# Expert
+- name: Package Expert
+  color: "#207de5"
+  description: "Requires alot of knowledge of the package."
+- name: Package Intermediate
+  color: "#207de5"
+  description: "Requires some of knowledge of the package."
+- name: Package Novice
+  color: "#207de5"
+  description: "Requires little of knowledge of the package."
+# Priority
+- name: Priority High
+  color: "#5319e7"
+  description: "Rapid action required."
+- name: Priority Medium
+  color: "#5319e7"
+  description: "Non-urgent action required."
+- name: Priority Low
+  color: "#5319e7"
+  description: "Rapid action not required."
+- name: Question
+  color: "#d876e3"
+  description: "For issues that are questions. "
+  aliases: [question]
+# CI Control Labels
+- name: Run cron CI
+  color: "#ffffff"
+  description: "Run cron CI on this PR."
+- name: Run publish
+  color: "#ffffff"
+  description: "Run publish CI on this PR."
+- name: pre-commit.ci run
+  color: "#ffffff"
+  description: "Automatically fix pre-commit issues on this PR."
+- name: Solar Physics Knowledge
+  color: "#b90ac9"
+  description: "This requires some basic understanding of solar physics."
+- name: Stale
+  color: "#ffffff"
+  description: "The bot will close this PR after 6 months (if enabled on this repo)."
+  aliases: [stale]
+- name: Still Needs Manual Backport
+  color: "#f97fa4"
+  description: "This PR needs manually backporting."
+  aliases: [ManualBackport]
+- name: Tests
+  color: "#0052cc"
+  description: "Affects tests in some measure."
+- name: Upstream Fix Required
+  color: "#f77960"
+  description: "A fix needs to go upstream to another package."
```

Comment:
Do we want to consider a downstream label?

Reply:
I can't even remember the last time we used the one on core?

Reply:
Fair.